tokio-trace/tests/macros.rs

```diff
@@ -6,6 +6,22 @@ extern crate tokio_trace;
 // they're compile-time tests, so I want to get line numbers etc out of
 // failures, and producing them with a macro would muddy the waters a bit.
 
+#[test]
+fn span() {
```

Comment:
Thanks for adding this test! 

If you're up for it, it would also be nice to have tests in `tests/span.rs` that use mock subscribers to actually expect that the correct target and level are set, but I'm happy to merge this without that.

Reply:
A mock subscriber test has been added.